SFR/Assembly/inner_assembly/make_SFR_assembly_model.py

```diff
@@ -0,0 +1,209 @@
+ap.add_argument('-n', dest='n_axial', type=int, default=100, help='Number of cells in the Z direction')
+ap.add_argument('-e', '--shannon_entropy', action='store_true', help='Add Shannon entropy mesh')
+ap.add_argument('--height', dest='height_of_the_core', type=float, default=30.0, help='Height of the reactor core')
+
+# Parse the arguments
+arguments = ap.parse_args()
+
+arguments=ap.parse_args()
+
+N=arguments.n_axial
+height=arguments.height_of_the_core
+shannon_entropy=arguments.shannon_entropy
+
+
+###################################################################
+##########################   Materials     ########################
+###################################################################
+
+u235 = openmc.Material(name='U235')
+u235.add_nuclide('U235', 1.0)
+u235.set_density('g/cm3', 10.0)
+
+u238 = openmc.Material(name='U238')
+u238.add_nuclide('U238', 1.0)
+u238.set_density('g/cm3', 10.0)
+
+pu238 = openmc.Material(name='Pu238')
+pu238.add_nuclide('Pu238', 1.0)
+pu238.set_density('g/cm3', 10.0)
+
+pu239 = openmc.Material(name='U235')
+pu239.add_nuclide('Pu239', 1.0)
+pu239.set_density('g/cm3', 10.0)
+
+pu240 = openmc.Material(name='Pu240')
+pu240.add_nuclide('Pu240', 1.0)
+pu240.set_density('g/cm3', 10.0)
+
+pu241 = openmc.Material(name='Pu241')
+pu241.add_nuclide('Pu241', 1.0)
+pu241.set_density('g/cm3', 10.0)
+
+pu242 = openmc.Material(name='Pu242')
+pu242.add_nuclide('Pu242', 1.0)
+pu242.set_density('g/cm3', 10.0)
+
+am241 = openmc.Material(name='Am241')
+am241.add_nuclide('Am241', 1.0)
+am241.set_density('g/cm3', 10.0)
+
+o16 = openmc.Material(name='O16')
+o16.add_nuclide('O16', 1.0)
+o16.set_density('g/cm3', 10.0)
+
+sodium = openmc.Material(name='Na')
+sodium.add_nuclide('Na23', 1.0)
+sodium.set_density('g/cm3', 0.96)
+
+cu63 = openmc.Material(name='Cu63')
+cu63.set_density('g/cm3', 10.0)
+cu63.add_nuclide('Cu63', 1.0)
+
+Al2O3 = openmc.Material(name='Al2O3')
+Al2O3.set_density('g/cm3', 10.0)
+Al2O3.add_element('O', 3.0)
+Al2O3.add_element('Al', 2.0)
+
+helium = openmc.Material(name='Helium for gap')
+helium.set_density('g/cm3', 0.001598)
+helium.add_element('He', 2.4044e-4)
+
+#Material mixtures
+
+inner_fuel_material= openmc.Material.mix_materials([u235, u238, pu238, pu239, pu240, pu241, pu242, am241, o16],[0.0019, 0.7509, 0.0046, 0.0612, 0.0383, 0.0106, 0.0134, 0.001, 0.1181],'wo')
+cladding_material = openmc.Material.mix_materials([cu63,Al2O3], [0.997,0.003], 'wo')
+
+materials=openmc.Materials([inner_fuel_material,sodium,helium,cladding_material])
+materials.export_to_xml()
+
+
+###############################################################################
+# Define problem geometry
+
+# Create cylindrical surfaces
+
+
+
+# Create cylindrical surfacesfuel_or = openmc.ZCylinder(surface_id=1, r=0.943/2) 
+
+fuel_or = openmc.ZCylinder(surface_id=1, r=0.943/2) 
+clad_ir = openmc.ZCylinder(surface_id=2, r=0.973/2) 
+clad_or = openmc.ZCylinder(surface_id=3, r=1.073/2) 
```

Comment:
Not sure you need these `surface_id`s here. Also we should probably use variables to define the radii